mod-source-record-storage-server/src/main/java/org/folio/services/handlers/AbstractPostProcessingEventHandler.java

```diff
@@ -73,13 +74,15 @@ public abstract class AbstractPostProcessingEventHandler implements EventHandler
   private final KafkaConfig kafkaConfig;
   private final MappingParametersSnapshotCache mappingParamsCache;
   private final Vertx vertx;
+  private final RecordService recordService;
 
-  public AbstractPostProcessingEventHandler(final RecordDao recordDao, KafkaConfig kafkaConfig,
+  public AbstractPostProcessingEventHandler(final RecordDao recordDao, KafkaConfig kafkaConfig, RecordService recordService,
```

Comment:
To fully replace usage of RecordDao to use only RecordService is not possible, because its necessary big refactoring?

Reply:
I checked the ability to fully replace RecordDao with RecordService, mostly it requires to declare updateParsedRecord() method in the RecordService, made appropriate refactoring

Reply:
looks great, thanks